engine/schema/builder.ts

```diff
@@ -180,12 +180,15 @@ export const newSchemaBuilder = (initial: SchemaData): SchemaBuilder => {
       const canonical = canonicalFileWith(base, namespace);
```

Comment:
the change in this file was necessary to avoid typings being created to primitive values like `null` or `string`. Previously it was added as a `definition` and referenced in the other components which makes no sense since it isn't reused across them.